content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -315,7 +318,51 @@ Users of the REST API can sign CSRs by submitting an UPDATE request to the `stat
 subresource of the CSR to be signed.
 
 As part of this request, the `status.certificate` field should be set to contain the
-signed certificate.
+signed certificate. This field contains one or more PEM-encoded certificates.
+
+All PEM blocks must have the "CERTIFICATE" label, contain no headers,
+and the encoded data must be a BER-encoded ASN.1 Certificate structure
+as described in [section 4 of RFC5280](https://tools.ietf.org/html/rfc5280#section-4.1).
+
+Example certificate content:
+
+```shell
```

Comment:
nit: this isn't Bourne shell code.

(I can't make a GitHub suggestion for a line that starts with three backticks, otherwise I would)

Reply:
done